cfme/tests/control/test_smoke_control.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""This test contains necessary smoke tests for the Control."""
+import pytest
+
+from cfme.base import Server
+from cfme import control, test_requirements
+from utils.appliance.implementations.ui import navigate_to
+
+pytestmark = [
+    test_requirements.control,
+    pytest.mark.smoke,
+    pytest.mark.tier(2)
+]
+
+destinations = [
+    control.explorer.ControlExplorer.__name__,
+    control.simulation.ControlSimulation.__name__,
+    control.import_export.ControlImportExport.__name__,
+    control.log.ControlLog.__name__
+]
+
+control_explorer_accordions = [
+    "Policy Profiles",
+    "Policies",
+    "Events",
+    "Conditions",
+    "Actions",
+    "Alert Profiles",
+    "Alerts"
+]
+
+
+@pytest.fixture(scope="module")
+def control_explorer_view():
+    return navigate_to(Server, "ControlExplorer")
+
+
+@pytest.mark.parametrize("destination", destinations)
+def test_control_navigation(destination):
+    """This test verifies presence of destinations of Control tab.
+
+    Steps:
+        * Open each destination of Control tab.
+    """
+    navigate_to(Server, destination)
```

Comment:
**Optional**
Capture the view and add an assert against view.is_displayed for an explicit 'test' here.